indico/modules/events/models/events.py

```diff
@@ -669,6 +669,20 @@ def can_lock(self, user):
         """Check whether the user can lock/unlock the event"""
         return user and (user.is_admin or user == self.creator or self.category.can_manage(user))
 
+    def can_display(self):
+        """Check whether the user can display the event in the category.
+        To be replaced with new permission mechanism: can('display', self, session.user)."""
+        return self.visibility is None or self.can_manage(session.user)
```

Comment:
`self.visibility != 0` since any other visibility includes the current category as well

Reply:
Ok